src/RestApiClient.js

```diff
@@ -200,7 +196,6 @@ class RestApiClient {
   * Set the request body depending on type of data.
   *
   * @param  {FormData|Object} data
-  * @return {FormData|string}
```

Comment:
This is the correct `@return` param for this function.

Reply:
oh shoot I didn't see it said `return` - thanks @DFurnes !